src/sync.ts

```diff
@@ -240,6 +249,82 @@ export class SyncApi {
                 RoomMemberEvent.Membership,
             ]);
         });
+
+        // TODO: Should we just move this to `room.ts`?
```

Comment:
Might be fine. This is within the `registerStateListeners` function which seems pretty relevant 🤷